operator/v1/types_ingress.go

```diff
@@ -233,6 +233,68 @@ type LoadBalancerStrategy struct {
 	// +kubebuilder:validation:Required
 	// +required
 	Scope LoadBalancerScope `json:"scope"`
+	// aws provides configuration settings that are specific to AWS
+	// load balancers.
+	//
+	// If this field is empty, "Classic" is used as the load balancer
+	// type.
+	//
+	// +optional
+	AWS *AWSLoadBalancerParameters `json:"aws,omitempty"`
```

Comment:
Given the cloud specific nature of this, a structure that includes something like https://github.com/openshift/api/blob/master/config/v1/types_infrastructure.go#L131 may be in order.

So

```
type LoadBalancerStrategy struct {
    Scope
    PlatformLoadBalancerParameters PlatformLoadBalancerParameters
}

type PlatformLoadBalancerParameters struct{
    Type PlatformType `json:"type"`

    AWS *AWSLoadBalancerParameters `json:"aws,omitempty"`
}
```

my apologies if you had this before and I didn't pay attention to the aws-y nature of this.